### DISCUSSION TITLE
Some kind of videos and audio not supported

### DISCUSSION THREAD

**Discussion by Cyberiacyberdyne** [mifi/lossless-cut#1950] [Support]
Lossless cut is really nice software for video editing, but when I try 4k or other 1080p video editing, some videos audio not supported, please fix this problem, and try to build a setup file, Thanks

**Comment by mifi** [comment]
need to know more info about the particular files. but most likely you can convert it to a supported format from the File menu

**Reply by Cyberiacyberdyne** [reply]
Large mkv file's and 4k videos not supported sometimes audio and some Times video not supported, I'm already try convert to supported formats but there's one option to say slowest high quality encode , and it's really too much slower :(    other options are likely to fail or unsupported video codec or no audio ! Please make lossless cut perfect.. thanks
